### ISSUE TITLE
Can't spawn bosses

### ISSUE THREAD

**Issue by lunrik** [kangarko/Boss#336]
se puede generar <Boss Name> en esta ubicación < this message appears when i try to summon a boss via command or bossegg ... could you please help me? ;) 
- lunrik

**Comment by kangarko** [comment]
Please set your Locale to "en", restart the server and post me the new message. Also post me the console log in case there are any errors.

**Comment by Reactfail** [comment]
Hey, I am the server's owner and will continue this thread.
I have set the language to en and tried again. There are no errors in the console.
![image](https://user-images.githubusercontent.com/39224427/60351217-9d68aa00-99c5-11e9-8936-69883af31daa.png)

**Comment by kangarko** [comment]
There is some other plugin such as WorldGuard or Residence or other preventing you from spawning bosses there. You can also set Egg_Spawning.Check_Regions to false. Try spawning them on a different location.

**Comment by Reactfail** [comment]
Hey, I found the reason for the problem. We were using a plugin called "SpawnersOnly". This plugin should allow mobs only to be spawned through spawners and spawn eggs. Looks like the plugin didn't work properly, spawn eggs didn't work (not even normal spawn eggs). We have removed the "SpawnersOnly" plugin and will probably write a fix for it soon. The Boss plugin now works perfectly, thank you for your help.
